code/modules/holiday/spacemas.dm

```diff
@@ -437,7 +437,7 @@ var/static/list/santa_snacks = list(/obj/item/reagent_containers/food/drinks/egg
 	anchored = 1
 	layer = NOLIGHT_EFFECTS_LAYER_BASE
 	pixel_x = -64
-	plane = PLANE_BLACKNESS + 1
+	plane = PLANE_DEFAULT + 2
```

Comment:
Planes that aren't in the list of standard planes might cause issues in the future. PLANE_NOSHADOW_ABOVE or PLANE_DEFAULT with a higher layer might be preferrable but this is also okay for now. Just a thing to keep in mind if it ever breaks (note that PLANE_BLACKNESS + 1 was also bad/worse).